seimas/accounts/templates/accounts/login.html

```diff
@@ -35,6 +35,13 @@
       </form>
     </div>
     {% endfor %}
+   <div class="auth-providers">
+      <form method='post' action="/accounts/login/">
```

Comment:
Do not hardcode urls, use [built-in url tag](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#url).